docs/components/console/manage-clusters/manage-ip-allowlists.md

```diff
@@ -5,6 +5,10 @@ description: "If your organization works within Camunda's Enterprise plan, you c
 keywords: [whitelist, allowlist, ip whitelist, ip allowlist]
 ---
 
+:::note
+Connections from Web Modeler are automatically part of the allowlist.
```

Comment:
@theburi can we more generally say that some Camunda system (such as the Web Modeler, or monitoring system) will automatically be part of the allowlist? Or do we only focus on the web modeler part?

Reply:
I would be more specific to potentially preempt clarification questions. If we mention generic systems, I would expect this to generate questions about specifics.

Reply:
OK, then let's update it later with more specifics when they come.

Reply:
Resolving thread, thank you both!